test/data/account/delete_tenant.yaml

```diff
@@ -0,0 +1,26 @@
+---
+version: 1
+interactions:
+- request:
+    headers:
+      sub: ["bcdd0b29-123d-11e8-a8bc-b69930b94f5c"] # will be compared against the `sub` claim in the incoming request's token
+    url: http://tenant/api/tenant?remove=false
+    method: DELETE
+  response:
+    status: 204 No Content
+    code: 204
```

Comment:
hmm.. would have been nice to tie these to the Tenant Design somehow...

Reply:
yeah, but I'm not sure how to automate this, honestly. Generate some samples from the design ?